apps/cowswap-frontend/src/modules/tokensList/pure/TokenTags/index.tsx

```diff
@@ -70,12 +70,12 @@ function TagDescriptor({ tags, children }: { children?: React.ReactNode; tags: T
   return (
     <styledEl.TagContainer>
       {tags.map((tag) => (
-        <MouseoverTooltip key={tag.id} text={tag.description}>
+        <HoverTooltip wrapInContainer  key={tag.id} content={tag.description}>
```

Comment:
Nitpick: another extra space

```suggestion
        <HoverTooltip wrapInContainer key={tag.id} content={tag.description}>
```


Sorry, I cannot not see this 🙈